api/core/tools/provider/builtin/searchapi/searchapi.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from core.tools.errors import ToolProviderCredentialValidationError
+from core.tools.provider.builtin.searchapi.tools.google import GoogleTool
+from core.tools.provider.builtin_tool_provider import BuiltinToolProviderController
+
+
+class SearchAPIProvider(BuiltinToolProviderController):
```

Comment:
https://github.com/langgenius/dify/blob/e852a2163402b6c967852bf2dc40a1469db393a0/api/core/tools/provider/builtin/google/google.py#L26

In our new tools page, it introduces a function to set the tool's tags.  `_get_tool_labels` 
You should add this one as well.